lib/python/vdm/server/HTTPListener.py

```diff
@@ -1111,6 +1148,102 @@ def delete(username, database_id):
         DEPLOYMENT_USERS.remove(current_user[0])
         return jsonify({'status': 1, 'statusstring': "User Deleted"})
 
+class StartDatabaseAPI(MethodView):
+    """Class to handle request to start servers on all nodes of a database."""
+
+    @staticmethod
+    def put(database_id):
+        """
+        Starts VoltDB database servers on all nodes for the specified database
+        Args:
+            database_id (int): The id of the database that should be started
+        Returns:
+            Status string indicating if the database start requesst was sent successfully
+        """
+
+        # TODO: assume deployment file has been sync'ed already????
+        members = []
+        current_database = [database for database in DATABASES if database['id'] == database_id]
+        if len(current_database) == 0:
+            abort(404)
+        else:
+            members = current_database[0]['members']
+	if len(members) == 0:
+            return make_response(jsonify({'statusstring': 'No servers configured for the database'}),
+                                             500)
+
+	# Check if there are valid servers configured for all ids
+        for server_id in members:
+            server = [server for server in SERVERS if server['id'] == server_id]
+            if len(server) == 0:
+                return make_response(jsonify({'statusstring': 'Server details not found for id ' + server_id}),
+                                             500)
+	# Now start each server
+        __host_name__ = socket.gethostname()
+        __host_or_ip__ = socket.gethostbyname(__host_name__)
+	failed = False
+	server_status = {}
+        for server_id in members:
+            server = [server for server in SERVERS if server['id'] == server_id]
+	    curr = server[0]
+            try:
+		if (curr['hostname'] == __host_or_ip__):
+                    deploymentcontents = get_database_deployment(database_id)
+                    retcode = start_local_server(deploymentcontents)
+		    if (retcode == 0):
+		        server_status[curr['hostname']] = 'Success'
+	            else:
+		        failed = True
+		        server_status[curr['hostname']] = 'Error starting server'
+		else:
+	            url = 'http://' + curr['hostname'] + \
+	                  ':8000/api/1.0/databases/' + str(database_id) + \
```

Comment:
use multiline concatenation as in  https://github.com/VoltDB/pro/blob/master/tests/apptests/kvbenchmark/runapp.py#L150